src/core/types/format_type.ml

```diff
@@ -253,7 +255,8 @@ let internal_tracks =
 let track =
   {
     Type.t = Track;
-    constr_descr = "a source track";
+    constr_descr = "a track";
```

Comment:
"track" is used for two things in liquidsoap:
- a track = a song
- a track = a media contents (audio / video / midi / metadata / etc.)
I'd suggest calling _channel_ the second one to make the distinction clear. What do you think?

```suggestion
    constr_descr = "a channel";
```

Reply:
Yes but channel is also for audio. How about `a media track`?